code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -633,3 +635,74 @@
 	R.add_fingerprint(user)
 	qdel(held)
 	held = null
+
+/obj/structure/roller_rack
+	name = "roller bed rack"
+	desc = "A rack for holding collapsed roller beds."
+	icon = 'icons/obj/rollerbed.dmi'
+	icon_state = "holder"
+	var/list/held
+	var/initial_beds = 4
+
+/obj/structure/roller_rack/Initialize()
+	. = ..()
+	LAZYINITLIST(held)
```

Comment:
There's no point in it being a lazylist if you initialize it at Initialize, it might aswell be a normal list